### PR TITLE
tbtcv2 application management

### PR DESCRIPTION
**Pull request by KPrasch**

### CODE REVIEW

**Comment by jMyles** [comment]
:eyes:

File: nucypher_ops/cli/tbtcv2.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                                                     namespace=namespace,
+                                                     network=network,
+                                                     envvars=envvars,
+                                                     cliargs=cliargs,
+                                                     resource_name='tbtcv2')
+
+    hostnames = deployer.config['instances'].keys()
+    if include_hosts:
+        hostnames = include_hosts
+    for name, hostdata in [(n, d) for n, d in deployer.config['instances'].items() if n in hostnames]:
```
Comment (jMyles) [inline nucypher_ops/cli/tbtcv2.py:32]: Why?  What happened to my comrade in verbosity and comprehensibility?  

https://www.getyarn.io/yarn-clip/4cfd97ad-d2a1-4142-b776-f438ebf46a88

Comment (KPrasch) [reply nucypher_ops/cli/tbtcv2.py:32]: I intend to clean and dry these entry points up, this is just some copypasta. ask @damonC! 😜

**Comment by derekpierre** [comment]
Nice work @KPrasch

File: nucypher_ops/ops/fleet_ops.py [hunk 2]
```diff
@@ -273,27 +260,33 @@ def chain_name(self):
...
+        return defaults
+
+    def update_generate_inventory(self, node_names, generate_keymaterial=False, **kwargs):
 
+        # filter out the nodes we will not be dealing with
+        nodes = {key: value for key,
+                 value in self.config['instances'].items() if key in node_names}
+        if not nodes:
+            raise KeyError(
+                f"No hosts matched the supplied names: {node_names}.  Try `nucypher cloudworkers list-hosts` or create new hosts with `nucypher-ops nodes create`")
```
[the remark this answers is not in the document]
Comment (vepkenez) [reply nucypher_ops/ops/fleet_ops.py:287]: heh.  oops!

[the remark this answers is not in the document]
Comment (derekpierre) [reply nucypher_ops/ops/fleet_ops.py:287]: No worries. I'll fix in PR #19 .

**Pull request merged**